projects/packages/google-fonts-provider/src/class-google-fonts-provider.php

```diff
@@ -0,0 +1,343 @@
+<?php
+/**
+ * WordPress webfonts provider for Google Fonts
+ *
+ * @package automattic/google-fonts-provider
+ * @since $$next-version$$
+ */
+
+namespace Automattic\Jetpack\Fonts;
+
+if ( ! class_exists( '\WP_Webfonts_Provider' ) ) {
```

Comment:
If the class doesn't exist, we return early?

Reply:
This is simply a way to stop loading the code in this file if the `WP_Webfonts_Provider` class isn't available (from Gutenberg/WP).